.github/workflows/python-package.yml

```diff
@@ -22,6 +22,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install 'numpy<1.20' 'scipy<1.6'
```

Comment:
Is there any way to only run this line if we are using py3.6?

Reply:
Changed!